docs/platforms/kotlin-multiplatform/troubleshooting.mdx

```diff
@@ -0,0 +1,47 @@
+---
+title: Troubleshooting
+description: "Troubleshoot and resolve edge cases."
+sidebar_order: 9000
+---
+
+If you need help solving issues with your Sentry Kotlin Multiplatform SDK, you can read the edge cases documented below. If you need additional help, you can [ask on GitHub](https://github.com/getsentry/sentry-kotlin-multiplatform/issues/new/choose). Customers on a paid plan may also contact support.
+
+## Tests not working
+
+If you configured the Sentry Kotlin Multiplatform SDK and tests are still failing with the following error: `ld: framework 'Sentry' not found` then follow these steps for the workaround:
+
+### 1. Download the Sentry xcframework
+
+Choose the release based on the version specified in the version compatibility table, download the `Sentry.xcframework.zip`, and unzip it.
```

Comment:
```suggestion
1. Ensure you have a compatible Sentry xcframework. Choose the release based on the version specified in the version compatibility table, download the `Sentry.xcframework.zip`, and unzip it.
```
I would consider reformatting the headings into ordered list items, since the instructions are each one step and aren't long enough to benefit much from clickable headings. If we add more troubleshooting items later, I think it'll also look cleaner.